entries/r/royalholloway.ac.uk.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Royal Holloway, University of London": {
+    "domain": "www.royalholloway.ac.uk",
```

Comment:
```suggestion
    "domain": "royalholloway.ac.uk",
```

Reply:
The correct domain is with www. there is no domain usage without.

Reply:
The domain is `royalholloway.ac.uk`. Whether or not a webserver responds to HTTP(S) requests to the domain is irrelevant.
If you want the entry to point to a specific URI use the `url` element, but the `domain` element should only have the domain without any subdomains and must have the same value as the file name.
At the moment your file name is `royalholloway.ac.uk` and domain value `www.royalholloway.ac.uk` causing the build to fail.